scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -59,6 +59,29 @@
                             order: Number.MIN_VALUE,
                             visibilityWindow: 300000000
                       },
+                      {% for wtrack in rhocall_wig_tracks %}
+                      {
+                        type: "wig",
+                        name: '{{ wtrack.name }}',
+                        url: '{{ url_for("alignviewers.remote_static", file=wtrack.url) }}',
+                        format: 'bigwig',
```

Comment:
Those are by default wig rather than bigwig. I suppose we can - and perhaps must for performance convert them - but perhaps make them wig for now.